src/ralph/models/converter.py

```diff
@@ -24,18 +21,50 @@ class ConversionItem:
     transformers: tuple[Callable[[Any], Any]]
 
     def __init__(self, dest: str, src=None, transformers=lambda _: _):
+        """Initializes ConversionItem.
+
+        Args:
+            dest (str): The destination path where to place the converted value.
+            src (str or None): The source from where the value to convert is fetched.
+                - When `src` is a path (ex. `context__user_id`) - the value is the item
+                    of the event at the path.
+                - When `src` is `*` - the value is the whole event.
+                - When `src` is `**` - the value is the original event string.
+                - When `src` is `None` - the value is `None`.
+            transformers (function or tuple of functions): The function(s) to apply on the source
+                value.
+        """
+
         object.__setattr__(self, "dest", tuple(dest.split(MODEL_PATH_SEPARATOR)))
-        src = tuple(src.split(MODEL_PATH_SEPARATOR)) if src else None
+        special = [None, "*", "**"]
+        src = src if src in special else tuple(src.split(MODEL_PATH_SEPARATOR))
         object.__setattr__(self, "src", src)
         transformers = tuple([transformers]) if callable(transformers) else transformers
         object.__setattr__(self, "transformers", transformers)
 
-    def get_value(self, event: dict):
-        """Returns fetched source value after having applied transformers to it."""
+    def get_value(self, event: dict, event_str: str):
```

Comment:
```suggestion
    def get_value(self, data: Union[dict, str]):
```

Reply:
Thanks) This simplifies the `get_value` method a lot) The only thing that bothers me here is that now the caller has to check the `conversion_item` `raw_input` field before calling `get_value` to pass the right value (ether dict or string).